fineract-provider/src/main/java/org/apache/fineract/infrastructure/event/external/service/message/MessageFactory.java

```diff
@@ -73,4 +97,9 @@ private String getMessageCreatedAt() {
         OffsetDateTime createdAt = DateUtils.getOffsetDateTimeOfTenant();
         return createdAt.withOffsetSameInstant(ZoneOffset.UTC).toLocalDateTime().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
     }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        log.info("Message source set to " + sourceUUID);
```

Comment:
Instead of concatenating the 2 values, use log message placeholders.
(`log.info("Message source set to {}", sourceUUID);`)